packages/ember-simple-auth/tests/simple-auth/mixins/application-route-mixin-test.js

```diff
@@ -44,6 +44,7 @@ describe('ApplicationRouteMixin', function() {
       });
 
       it("translates the session's 'sessionAuthenticationSucceeded' event into an action invocation", function(done) {
+        expect(this.route.get('_authRouteEntryComplete')).to.be.true;
```

Comment:
Sorry for being so finicky, I actually really appreciate the PR, but this should have it's own test…